### DISCUSSION TITLE
zigbee2mqtt using a ZBDongle-P (Sonoff Zigbee 3.0) on Home Assistant Yellow

### DISCUSSION THREAD

**Discussion by marothe** [Koenkk/zigbee2mqtt#18145] [Devices]
Hey all,

I hope this is the right place to ask, if not, please point me to the right place. 

I have a HA Yellow and have been using Z2M via a ZZH! dongle. That stopped working a while ago and I think I broke it while flashing, so I got a Sonoff Zigbee 3.0 USB dongle in the mail today ( it says ZBDongle-P on the back...)

Before I decided to order this new dongle, I tried the internal zigbee antenna of the HA yellow, as it is at least listed as experimental in the documentation. This was my config:

`{
  "data_path": "/config/zigbee2mqtt",
  "socat": {
    "enabled": false,
    "master": "pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777",
    "slave": "tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5",
    "options": "-d -d",
    "log": false
  },
  "mqtt": {
    "server": "mqtt://192.XXX.XXX.XX:1883",
    "user": "thisisauser",
    "password": "thisisapassword"
  },
  "serial": {
    "port": "/dev/ttyAMA1",
    "baudrate": 115200,
    "adapter": "ezsp",
    "rtscts": false
  }
}`

Now, plugging in the new dongle it gives me /ttyUSB0:

![image](https://github.com/Koenkk/zigbee2mqtt/assets/29024610/d2495152-a0c1-41f3-bb36-c8663a1f78f6)

I am uanble to see the internal silicon labs chip from the HA yellow anymore? but I change it to 

`adapter: /dev/ttyUSB0` Z2M does not start. It gives me the following error:

> Error: Reset error: Error: {"sequence":-1} after 10000ms at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:299:23 at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)

Therefore....I am quite confused....HA Yellow sees the Sonoff dongle but I am unable to use it, only if I use my "old" config for the internal zigbee antenna I am able to start Z2M and pair devices.

I did disable ZHA in the integrations part of the settings, see here:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/29024610/542b169f-9a9b-40ae-bbd0-b107b3362fcb)

Any pointers maybe? I have the feeling I forgot to change something back or else, but reinstalling the add-on did not help...thanks in advance!

**Comment by kasperkc87** [comment]
I have a similar issue, I am trying to get Z2M to work on HA Yellow, with my external Texas Instruments LAUNCHXL-CC1352P-2 adapter

I am able to discover the device, but Z2M fails to connect. I tried all the things mentioned [here](https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html)

I get the error: `SRSP - SYS - ping after 6000ms`

**Comment by marothe** [comment]
@kasperkc87 Hey, I solved the problem here: https://community.home-assistant.io/t/zigbee2mqtt-using-a-zbdongle-p-sonoff-zigbee-3-0-on-home-assistant-yellow/587003/5

But, I am pretty sure your problem does have a different cause. Sorry I cannot help you.
